src/main/resources/view/WebView.css

```diff
@@ -0,0 +1,15 @@
+/**
+ * WebView is unable to load '-apple-system', and 'San Francisco' (the default macOS
+ * font) cannot be called by name. Thus, we call '.SFNSText-Regular' on failure.
+ */
+body {
+    font-family: -apple-system,
```

Comment:
Left `-apple-system` in so that if `WebView` ever gets fixed, everything starts working normally again.